app/views/admin/cases/show.html.slim

```diff
@@ -0,0 +1,148 @@
+
+= javascript_pack_tag 'tribute'
+= stylesheet_pack_tag 'tribute'
+= javascript_pack_tag 'admin_case'
+
+
+.row.p-3.mb-4.shadow-sm.mx-1.rounded
+  .col-4
+    span.mr-2.text-dark Publisher Email:
+    span.font-weight-bold= link_to(@case.publisher.email, admin_publisher_path(@case.publisher), target: "_blank")
+
+  .col-4
+    span.mr-2.text-dark Contributions:
+    span.font-weight-bold
+      = publisher_contribution_bat_balance(@case.publisher)
+      small.text-muted.font-weight-bold= " BAT"
+
+      small.text-muted
+        span.mx-2= " — "
+        =publisher_bat_percent(@case.publisher)[:contribution]
+
+  .col-4
+    span.mr-2.text-dark Referrals:
+    span.font-weight-bold
+      = publisher_referral_bat_balance(@case.publisher)
+      small.text-muted.font-weight-bold= " BAT"
+      small.text-muted
+        span.mx-2= " — "
+        =publisher_bat_percent(@case.publisher)[:referrals]
+
+.row.mb-4
+
+
+  .col-10
+    .c-4.rounded.p-3.mb-p5.shadow
+      h3
+        span= "Case ##{@case.id.last(5)}"
```

Comment:
Might be worth doing ascending numerically like 1, 2, 3 given the possiblity of collision and trying to replicate Github

Reply:
https://github.com/brave-intl/publishers/pull/2027/commits/b28066ee21b63308a034dc68eeeb1b25bb9f1827